Jenkinsfile

```diff
@@ -237,6 +237,11 @@ pipeline {
         }
       }
     }
+    stage('Log check') {
+      steps {
+        writeFile file: "jenkins_output.log", text: containerLog(name: "jnlp", returnLog: true)
+        findText fileSet: "jenkins_output.log", regexp: '.*EPIPE.*', unstableIfFound: true
```

Comment:
Do we need to write out the file here - since we're running in the jnlp container can't we just do `findText` with `alsoCheckConsoleOutput: true` instead of a `fileSet`?

Reply:
Updated to use `alsoCheckConsoleOutput` instead of `fileSet` in [b230f9c](https://github.com/IBM/couchbackup/pull/708/commits/b230f9ca8445b52003e69a005dd4ca46ed88b95f)